workflow-templates/psalm.yml

```diff
@@ -0,0 +1,30 @@
+name: Static analysis
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - main
+      - stable*
+
+jobs:
+  static-analysis:
+    runs-on: ubuntu-latest
+
+    name: Nextcloud ${{ matrix.ocp-version }}
```

Comment:
Where is the matrix?

Also it would be good to have the same header as other workflows

Reply:
Yeah I noticed the other day, but didn't have time to fix